package.json

```diff
@@ -30,13 +28,21 @@
   "devDependencies": {
     "babel": "^5.5.8",
     "babel-eslint": "^3.1.15",
+    "babel-loader": "^5.1.4",
     "dependency-check": "^2.4.0",
     "eslint": "^0.23.0",
-    "faucet": "0.0.1",
-    "istanbul": "^0.3.15",
+    "isparta-loader": "^0.2.0",
+    "jasmine": "^2.3.1",
```

Comment:
Nope. Karma just got even more expensive. **Karma is vetoed**

Can you make this happen without Karma? If not, I'll close this PR.